lib/ttfunk/subset/code_page.rb

```diff
@@ -55,7 +55,7 @@ def includes?(character)
       end
 
       def from_unicode(character)
```

Comment:
I don't know the code base well but if this method could cache its return values, another 27 million object allocation could be removed, bringing down the total allocated objects to the same level as Prawn + built-in PDF font.

Reply:
This can definitely be cached, the mapping between codepage characters and Unicode characters is static.

Reply:
Thanks for the answer - I will change this.